shims/hdp25/assemblies/hdp25-shim/src/assembly/assembly.xml

```diff
@@ -10,7 +10,7 @@
       <outputDirectory>/</outputDirectory>
     </fileSet>
     <fileSet>
-      <directory>${oss.license.directory}</directory>
+      <directory>${oss.license.subdirectory}</directory>
```

Comment:
I replaced ${oss.license.directory} property with ${oss.license.unpack.directory} and added one new property ${oss.license.subdirectory} - this is a place where now OSS license html file is (i.e. target\oss_license\oss-licenses-7.1-SNAPSHOT\licenses\PentahoHadoopShim_cdh59_OSS_Licenses.html)